### ISSUE TITLE
Contrast ratio review

### ISSUE THREAD

**Issue by martgnz** [PopulateTools/gobierto#461]
We should review the contrast ratio of every page to meet WCAG 2, while maintaining our branding:
- Text should be 4.5:1.
- Headlines, 3:1.

We can exclude disabled UI elements and logos.

## Homepage
![screen shot 2017-03-03 at 15 58 55-fullpage](https://cloud.githubusercontent.com/assets/1236790/23555741/5f2b6df4-002a-11e7-8cce-adc195b035ed.png)

## Consultations
![screen shot 2017-03-03 at 16 00 34-fullpage](https://cloud.githubusercontent.com/assets/1236790/23555808/9a80a0a4-002a-11e7-9bbb-e2c887f8672a.png)

## Cargos y agendas
![screen shot 2017-03-03 at 16 02 36-fullpage](https://cloud.githubusercontent.com/assets/1236790/23555891/df4c8a22-002a-11e7-81ad-dafb03ee24be.png)
